src/FlowPlayer.cpp

```diff
@@ -75,6 +75,11 @@ static void migrateCache()
         }
         QDir(QStandardPaths::writableLocation(QStandardPaths::GenericCacheLocation)).rmdir("flowplayer");
     }
+    // if the media-art directory does not exist, make it.
+    if (!QFileInfo(newCache).isDir()) {
+        QDir().mkpath( newCache );
```

Comment:
According to documentation, `mkpath` craetes the parent directories if they don't exist already. This line is not necessary.